### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by nameolya**

### CODE REVIEW

File: src/service-worker.ts [hunk 1]
```diff
@@ -1,80 +0,0 @@
...
-// You can also remove this file if you'd prefer not to use a
-// service worker, and the Workbox build step will be skipped.
-
-import { clientsClaim } from 'workbox-core';
-import { ExpirationPlugin } from 'workbox-expiration';
-import { precacheAndRoute, createHandlerBoundToURL } from 'workbox-precaching';
-import { registerRoute } from 'workbox-routing';
-import { StaleWhileRevalidate } from 'workbox-strategies';
-
-declare const self: ServiceWorkerGlobalScope;
```
Comment (mgorski-mg) [inline src/service-worker.ts:17]: Our idea was to have this ServiceWorker to make our app Progressive Web App and make it able to be installed like normal app.

**Review by mgorski-mg — APPROVED** [review]
Looks like there is so much not used code, probably from copy pasting from the other repo as you wrote :)
But we need to make sure to use what we need and remove the rest. It's ok to leave things because it will be needed for next PR, but I feel there are some files not needed at all. But I'm not FE, so I might be wrong 😅

**Comment by nameolya** [comment]
> Looks like there is so much not used code, probably from copy pasting from the other repo as you wrote :) But we need to make sure to use what we need and remove the rest. It's ok to leave things because it will be needed for next PR, but I feel there are some files not needed at all. But I'm not FE, so I might be wrong 😅

@mgorski-mg I agree. But I would keep as many components as possible because the design we have now is not final and we may need something in the future. We can clean it up later

**Comment by mgorski-mg** [comment]
> > Looks like there is so much not used code, probably from copy pasting from the other repo as you wrote :) But we need to make sure to use what we need and remove the rest. It's ok to leave things because it will be needed for next PR, but I feel there are some files not needed at all. But I'm not FE, so I might be wrong 😅
> 
> @mgorski-mg I agree. But I would keep as many components as possible because the design we have now is not final and we may need something in the future. We can clean it up later

Sure! :)

File: src/service-worker.ts (hunk 1 above)
Comment (nameolya) [reply src/service-worker.ts:17]: reverted

**Pull request merged**